bounties/0019 Proof of Attendance Infrastructure.md

```diff
@@ -32,3 +32,5 @@ Basic requirements:
 | 1   | Create API library for NFT Minting and distrubtion | Complete basic requirements mentioned above with good documentation and tests.                                                                 | $3,000                   |
 | 2   | Event attendees lookup                             | Add a way for people to look up wallets with the same event NFTs.                                                                              | $1,000                   |
 | 3   | Event NFT ownership verification                   | Add an API for people to verify the event NFT - using their signiture to prove that they are the owner of the account associated with the NFT. | $2,000                   |
+| 4  | Deployed no-code tool for non-technical event organizers                   | Add a frontend (preferably a form) with wallet intergration (XUMM) for non-technical event organizers to mint and send event NFT's, the website should be deployed publicly | $1,500                   |
```

Comment:
Typo
```suggestion
| 4  | Deployed no-code tool for non-technical event organizers                   | Add a frontend (preferably a form) with wallet integration (XUMM) for non-technical event organizers to mint and send event NFT's, the website should be deployed publicly | $1,500                   |
```

Reply:
Should the website deployment details be included in the repo (i.e. in the README?)

Reply:
"and send event NFT's".

If we have event with N as number of people, we (as organiser), would probably like to mint N NFTs. Correct?
And to distribute it, it would be probably done by uploading some type of document (csv file or something like that), with list of addresses to send them this NFTs?

Reply:
to answer question 1: yes 
question 2: either a document or a form/link for individual accounts since realistically the event organizers can distribute NFT's at in-person events where attendees check in individually.